OPHD/States/MapViewStateHelper.cpp

```diff
@@ -278,11 +278,9 @@ bool selfSustained(StructureID id)
 	case StructureID::SID_COMM_TOWER:
 	case StructureID::SID_ROAD:
 		return true;
-		break;
 
 	default:
 		return false;
-		break;
 	}
```

Comment:
Another way this could be written, and easily extended, is to use a `std::array` to hold the IDs of self sustained structures, then use `std::find` to check if the ID exists in the self sustained array. The array could be `constexpr` defined in the cpp file, and the method could become a one-liner:
```cpp
return std::find(selfSustainedIds.begin(), selfSustainedIds.end(), id) != selfSustainedIds.end();
```

You could even mark the function as `constexpr`, allowing it to be evaluated at compile time (when passed a literal constant, rather than a variable).

Reply:
True.

If this case statement starts to get larger (before it was a simple if structureID == Comm Tower), that would be the solution I opt for.